### ISSUE TITLE
markFieldsIndicator:MarkMandatoryFields only applies at form creation

### ISSUE THREAD

**Issue by mistyn8** [umbraco/Umbraco.Forms.Issues#660]
Was expecting this to override any form that wasn't explicitly altered to be `noIndicator` via the form settings.
much like the defaultTheme.

I'm guessing that this isn't possible as once a form is created with defaults the `noIdicator` setting is set.
Maybe the documentation should mention the limitation? 

Or better have a null state on the form setting that would then fall back to the default?

**Comment by AndyButland** [comment]
Yes, that's right - all these default form settings are used as the value to use when a new form is created.  Once it's been created, the values have been set on the form, and subsequent changes to the defaults in configuration won't change the values of existing forms.

Documentation for these settings are [here](https://our.umbraco.com/documentation/Add-ons/UmbracoForms/Developer/Configuration/#form-default-settings-configuration), to which I'll add a short update to to make this clearer.  PR for this update is [here](https://github.com/umbraco/UmbracoDocs/pull/3566).
